pkg/event/registry_test.go

```diff
@@ -0,0 +1,106 @@
+package event_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	k8sV1 "k8s.io/api/core/v1"
+	v1meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/submariner-io/submariner/pkg/event"
+	"github.com/submariner-io/submariner/pkg/event/logger"
+)
+
+const npOVNKubernetes = "OVNKubernetes"
+const npGenericKubeproxyIptables = "GenericKubeproxyIptables"
+
+var _ = Describe("Event Registry", func() {
+	When("Registry is populated with different network plugin handlers", func() {
+		var (
+			ovn, generic *TestHandler
+			registry     event.Registry
+		)
+
+		BeforeEach(func() {
+			registry = event.NewRegistry("test-registry", npGenericKubeproxyIptables)
+			ovn = NewTestHandler("handler-ovn", npOVNKubernetes)
+			generic = NewTestHandler("handler-ovn", npGenericKubeproxyIptables)
+			registry.AddHandlers(logger.NewHandler(), ovn, generic)
```

Comment:
nit: logger.NewHandler() is unused in this testcase.

Reply:
I wanted the logger to be exercised for all methods, so if there's something broken about it it will fail.

Reply:
I think that should be in a separate test specific to the logger handler.